app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,201 @@
+/*
+* This is a manifest file that'll be compiled into application.css, which will include all the files
+* listed below.
+*
+* Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
+* or any plugin's vendor/assets/stylesheets directory can be referenced here using a relative path.
+*
+* You're free to add application-wide styles to this file and they'll appear at the bottom of the
+* compiled file so the styles you add here take precedence over styles defined in any styles
+* defined in the other CSS/SCSS files in this directory. It is generally better to create a new
+* file per style scope.
+*
+*= require bootstrap_and_overrides
+*= require chosen
+*= require_tree .
+*= require_self
+*/
+
+//@import "bootstrap-sprockets";
+//@import "bootstrap";
+
+// Font Awesome
+@import "font-awesome-sprockets";
+@import "font-awesome";
+
+
+// Bootstrap body padding for fixed navbar
+body {
+  padding-bottom: 80px;
+}
+
+.navbar {
+  height: 60px;
+  background-color: #0E426C;
+  opacity: 0.80;
+  position: relative;
+  margin-bottom: 0px;
+
+  #logo img {
+    width: 100px;
+    margin-top: -27px;
+  }
+
+  .navbar-header{
+    height:59px;
+    }
+
+  ul.nav.navbar-nav{
+    margin-top: 0px;
+
+    .directory{
+      font-size: 2em;
+    }
+
+    li a{
+      height: 60px;
+      padding-top: 19px;
+    }
+
+    li a.dropdown-toggle.directory{
+      padding-top: 15px;
+    }
+
+    .divider{
+      margin:0
+    }
+  }
+}
+
+.org-container {
+  height: 200px;
+
+  #org-logo {
+    width: 60px;
+    height: 60px;
+  }
+}
+
+#mission-div1 {
+  display: block;
+}
+#mission-div2 {
+  display: block;
+}
+#mission-div3 {
+  display: block;
+}
```

Comment:
Refactor these mission divs into a class so that you can change them all with one style